### PR TITLE
Review.rst: Update review docs

### PR DESCRIPTION
**Pull request by monsij**
This commit reflects the recent changes how
the developers respond to pull requests.

Closes https://github.com/coala/documentation/issues/529

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by sangamcse** [comment]
You should have done this in previous PR https://github.com/coala/coala/pull/5199 😄

**Comment by monsij** [comment]
Yup that's true .Actually when I reset my previous commit(I should have amended the commit body),PR #5199 automatically got closed.

File: docs/Developers/Review.rst [hunk 1]
```diff
@@ -75,13 +77,23 @@ The review process for coala is as follows:
...
+
 
 4. If the commits are not linearly mergeable into master, rebase and go
    to step one.
 5. All commits are acknowledged and fit linearly onto master. All
    continuous integration services (as described below) pass. A maintainer
-   may leave the ``@rultor merge`` command to get the PR merged automatically.
+   may leave the ``@gitmate-bot rebase`` and ``@gitmate-bot ff``
+   command to get the branch mentioned in the PR rebased,fast-forwarded
+   and merged automatically.Thanks to gitmate-bot.
```
Comment (sangamcse) [inline docs/Developers/Review.rst]: A space after `.` and `,`.

**Comment by monsij** [comment]
@li-boxuan : Thanks for it.You got it right!

File: docs/Developers/Review.rst [hunk 2]
```diff
@@ -75,13 +77,23 @@ The review process for coala is as follows:
 
     .. note::
 
-        Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
+          Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
+
+
+   * In case a member is reviewing it:
+
+    - Looks good to me better known as ``LGTM`` in case the commit is ready
```
Comment (sangamcse) [inline docs/Developers/Review.rst]: It would be better if you put `Looks good to me` between inverted coma.

Comment (monsij) [reply docs/Developers/Review.rst]: Done!

File: docs/Developers/Review.rst (hunk 1 above)
Comment (monsij) [reply docs/Developers/Review.rst]: Done!

File: docs/Developers/Review.rst [hunk 3]
```diff
@@ -65,6 +65,8 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case an owner is reviewing it:
```
Comment (sangamcse) [inline docs/Developers/Review.rst]: Better to use `maintainer` instead of `owner`.
Otherwise LGTM 😄 
cc @Makman2

Comment (monsij) [reply docs/Developers/Review.rst]: But I think it is mentioned in the contributors page -- owner/member

Comment (monsij) [reply docs/Developers/Review.rst]: Done @sangamcse  🎉

File: docs/Developers/Review.rst [hunk 4]
```diff
@@ -75,13 +77,23 @@ The review process for coala is as follows:
 
     .. note::
 
-        Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
+          Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
```
Comment (kriti21) [inline docs/Developers/Review.rst]: Why have you added leading white spaces here ?

Comment (kriti21) [reply docs/Developers/Review.rst]: This change here is not required IMO

Comment (monsij) [reply docs/Developers/Review.rst]: Should it be mentioned after members review points?

Comment (kriti21) [reply docs/Developers/Review.rst]: Yes that would be better

Comment (monsij) [reply docs/Developers/Review.rst]: okay! 👍

File: docs/Developers/Review.rst [hunk 5]
```diff
@@ -65,6 +65,15 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case a member is reviewing it:
+
+    - ``Looks good to me`` better known as ``LGTM`` in case the commit is ready
+         to be merged,or
+    - Additional comments if something needs to be rectified.
+
```
Comment (kriti21) [inline docs/Developers/Review.rst]: Only one blank line would be sufficient here.

Comment (monsij) [reply docs/Developers/Review.rst]: Done!

**Comment by kriti21** [comment]
Failing builds. Please check the logs and improve :)

File: docs/Developers/Review.rst (hunk 5 above)
Comment (monsij) [reply docs/Developers/Review.rst]: @kriti21 : why removal of a new line is causing the build fail?

File: docs/Developers/Review.rst [hunk 6]
```diff
@@ -65,6 +65,14 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case a member is reviewing it:
+
+    - ``Looks good to me`` better known as ``LGTM`` in case the commit is ready
+         to be merged,or
```
Comment (jayvdb) [inline docs/Developers/Review.rst]: `merged,or` is incorrect grammar

Comment (monsij) [reply docs/Developers/Review.rst]: Please check if it's correct now @jayvdb

**Comment by monsij** [comment]
Please review @kriti21

File: docs/Developers/Review.rst [hunk 7]
```diff
@@ -65,6 +65,14 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case a member is reviewing it:
+
+    - ``Looks good to me`` better known as ``LGTM`` in case the commit is
+      ready, or
+    - Additional comments if something needs to be rectified.
```
Comment (adtac) [inline docs/Developers/Review.rst]: Remove Additional, it's either LGTM or it's not.

File: docs/Developers/Review.rst [hunk 8]
```diff
@@ -65,6 +65,14 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case a member is reviewing it:
```
Comment (adtac) [inline docs/Developers/Review.rst]: hmm, maybe "newcomer" is better than member

File: docs/Developers/Review.rst (hunk 7 above)
Comment (monsij) [reply docs/Developers/Review.rst]: I think it's not like that. Members do provide additional information regarding the required changes to be made.

File: docs/Developers/Review.rst (hunk 8 above)
Comment (monsij) [reply docs/Developers/Review.rst]: There are only two distinctions as per coala : member or maintainer. I don't think any change is required. @adtac

File: docs/Developers/Review.rst (hunk 7 above)
Comment (monsij) [reply docs/Developers/Review.rst]: Oh sorry, I got it now. Thanks for it 🎉

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5200/commits/ef98c11e2ebc99baf87fdda8502dcb36af76bcc6

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**